src/index.tsx

```diff
@@ -121,6 +122,10 @@ const router = createBrowserRouter([
     path: '/downloads',
     element: <Downloads />,
   },
+  {
+    path: '/processTable',
```

Comment:
same here, `BillDetails` sounds better

Reply:
path, or urls should have hypen instead of spaces 

`proccess table` => `proccess-table`